packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -0,0 +1,147 @@
+import BadRequestError from "@metriport/core/util/error/bad-request";
+import { MetriportError } from "@metriport/core/util/error/metriport-error";
+import axios, { AxiosInstance, AxiosResponse } from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { STU3Bundle, stu3BundleSchema } from "../models/bundle";
+import { OrganizationList } from "../models/organization";
+
+dayjs.extend(duration);
+
+const DEFAULT_AXIOS_TIMEOUT = dayjs.duration(120, "seconds").asMilliseconds();
+const MAX_COUNT = 1000;
+const JSON_FORMAT = "_format=json";
+
+export enum APIMode {
+  dev = "dev",
+  staging = "stage",
+}
+
+/**
+ * This SDK operates on FHIR STU3 format.
+ */
+export class Carequality {
+  private static readonly devUrl = "https://dev-dir-ceq.sequoiadns.org/fhir-stu3/1.0.1";
+  private static readonly stagingUrl = "https://stage-dir-ceq.sequoiaproject.org/fhir-stu3/1.0.1";
+
+  // NOTE: These URLs can be used if we migrate to FHIR R4 format.
+  // private static readonly devUrl = "https://directory.dev.carequality.org/fhir";
+  // private static readonly  stagingUrl = "https://directory.stage.carequality.org/fhir";
+
+  static ORG_ENDPOINT = "/Organization/";
+  readonly api: AxiosInstance;
+  readonly apiKey: string;
+
+  /**
+   * Creates a new instance of the Carequality API client pertaining to an
+   * organization to make requests on behalf of.
+   *
+   * @param apiKey          The API key to use for authentication.
+   * @param apiMode         The mode the client will be running.
+   * @param options         Optional parameters
+   * @param options.timeout Connection timeout in milliseconds, default 120 seconds.
+   */
+  constructor(apiKey: string, apiMode: string, options: { timeout?: number } = {}) {
```

Comment:
`apiMode` should be of type `APIMode`, right?

I would also make it optional, with `staging` being the default now and later we make prod the default, but that's not a big deal.